apps/vr-tests-web-components/package.json

```diff
@@ -12,9 +12,6 @@
     "type-check": "tsc -p . --baseUrl . --noEmit",
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32308&commitId=efb13f18e3475058d7d18b004dec69fb57843052&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|6| Changed|
|Avatar Converged.badgeMask.chromium.png|4| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1136| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|975| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>